src/scss/_mixins.scss

```diff
@@ -1,18 +1,39 @@
-@mixin button {
+@mixin fonts-style($font-size: 16px, $font-weight: 500, $line-height: 26px, $color: $basic-state-text) {
+  color: $color;
+  font-weight: $font-weight;
+  font-size: $font-size;
+  line-height: $line-height;
+}
+
+@mixin button ($width: 110px, $height: 46px, $border-radius: 3px,) {
+  @include fonts-style(16px, 700, 24px, $basic-state-text);
   font-family: 'Poppins', sans-serif;
```

Comment:
Please add this property to font-style and create a variable for its value

Reply:
You can provide this property only once in common styles and it will work for the whole project.